Source/SwiftLintFramework/Rules/EmptyStringRule.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  EmptyStringRule.swift
+//  SwiftLint
+//
+//  Created by Davide Sibilio on 02/22/18.
+//  Copyright © 2015 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct EmptyStringRule: ConfigurationProviderRule, OptInRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "empty_string",
+        name: "Empty String",
+        description: "Prefer checking `isEmpty` over comparing `string` to \"\".",
```

Comment:
what do you think about changing this to `"Prefer checking `isEmpty` over comparing `string` to an empty string literal`?"

Reply:
I think that is better